.travis.yml

```diff
@@ -6,17 +6,40 @@ before_install:
   - pip install --user awscli
 install:
   - yarn install
-script:
-  - yarn validate
+stages:
+  - name: "Test"
+  - name: "Deploy"
+    if: tag IS present AND env(TRAVIS_NODE_VERSION) IS "10.16.3"
+jobs:
+  include:
+    - stage: "Test"
+      name: "lint"
+      script: "yarn lint"
+    - name: "cli: unit tests"
+      script: "yarn workspace zapier-platform-cli test"
+    - name: "cli: smoke tests"
+      script: "yarn workspace zapier-platform-cli smoke-test"
+    - name: "core: unit tests"
+      script: "yarn workspace zapier-platform-core test"
+    - name: "core: smoke tests"
+      script: "yarn workspace zapier-platform-core smoke-test"
+    - name: "schema: unit tests"
+      script: "yarn workspace zapier-platform-schema test"
+    - name: "schema: smoke tests"
+      script: "yarn workspace zapier-platform-schema smoke-test"
+    - name: "legacy-scripting-runner: integration tests"
+      script: "yarn workspace zapier-platform-legacy-scripting-runner test"
+    - stage: "Deploy"
+      script: skip
+      deploy:
+        provider: script
+        script: ./scripts/publish.sh
+        on:
+          tags: true,
+          node: "10.16.3"
```

Comment:
This is probably fine as is, but we might consider using `10.x` if it's available (or `12.x`, soon!) since lambda doesn't stick to a single version anymore.